package/android/src/main/java/com/reactnativecommunity/slider/ReactSliderEvent.java

```diff
@@ -46,9 +46,10 @@ public String getEventName() {
   public short getCoalescingKey() {
     return 0;
   }
+
   @Override
-  public void dispatch(RCTEventEmitter rctEventEmitter) {
-    rctEventEmitter.receiveEvent(getViewTag(), getEventName(), serializeEventData());
+  public void dispatchModern(RCTModernEventEmitter rctEventEmitter) {
+      rctEventEmitter.receiveEvent(getViewTag(), getEventName(), serializeEventData());
```

Comment:
Extra and unnecessary diff - please remove the incorrect indent and keep 2 spaces.